softpack_core/schemas/environment.py

```diff
@@ -481,11 +481,7 @@ def create(cls, env: EnvironmentInput) -> CreateResponse:  # type: ignore
         if not isinstance(response, CreateEnvironmentSuccess):
             return response
 
-        environment = Environment.get_env(Path(env.path), env.name)
-        if environment is not None:
-            cls.insert_new_env(environment)
-        else:
-            return EnvironmentNotFoundError(path=env.path, name=env.name)
```

Comment:
Isn't this a behaviour change that we no longer return an error if not found? Or can't this actually happen?

Reply:
It cannot happen as we literally have just created it and checked the return value to confirm.